sagemaker-pipeline-parameterization/notebook.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        JsonGet,
</pre><p>This version of JsonGet is deprecated. Switch to <code>sagemaker.workflow.functions.JsonGet</code>. ﻿You'll then have to rename the JsonGet <code>step</code>  parameter to <code>step_name</code>  .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3021/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>